drivers/bluetooth/hci/Kconfig.esp32

```diff
@@ -16,4 +15,12 @@ choice ESP_BT_HEAP
 
 endchoice # ESP_BT_HEAP
 
-endif
+config ESP32_BT_CONTROLLER_STACK_SIZE
+	int "Bluetooth controller stack size"
+	default 4096
+
+config ESP32_BT_CONTROLLER_TASK_PRIO
```

Comment:
I think this symbol should be ranged between the priorities level supported by zephyr.

An invalid setting here may result in a silent failure.